website/docs/r/codebuild_webhook.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "aws"
+page_title: "AWS: aws_codebuild_webhook"
+sidebar_current: "docs-aws-resource-codebuild-webhook"
+description: |-
+  Provides a CodeBuild Webhook resource.
+---
+
+# aws_codebuild_webhook
+
+Provides a CodeBuild Webhook resource.
+
+~> **Note:** The AWS account that Terraform uses to create this resource *must* have authorized CodeBuild to access GitHub's OAuth API. This is a manual step that must be done *before* creating webhooks with this resource. If OAuth is not configured, AWS will return an error similar to `ResourceNotFoundException: Could not find access token for server type github`.
```

Comment:
This manual step must also be done in every applicable region where the resource will be used. We should probably also point to the AWS documentation surrounding project webhooks here. Will update on merge.